tests/spec/core/xref-spec.js

```diff
@@ -65,14 +65,24 @@ describe("Core — xref", () => {
         <p>As <a data-cite="!infra">ASCII uppercase</a> is valid dfn, it resolves to fragment. a local data-cite (infra) overrides parent's datacite.</p>
 
         <p>As <a data-cite="!infra">ASCII uppercasing</a> doesn't exist in INFRA, it resolves to spec only.</p>
+
+        <!-- local links with empty parent data-cite -->
```

Comment:
This test might be getting a little big... would be good to split it, but have both conditions tested: that is, local link, and external link inside the section.